pkg/cmd/root.go

```diff
@@ -27,6 +29,11 @@ func NewCmd() *cobra.Command {
 		Short:                  "ike lets you safely develop and test on prod without a sweat",
 		BashCompletionFunction: completion.BashCompletionFunc,
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error { //nolint[:unparam]
+			if v.Released() && !version.IsLatestRelease() {
+				latestRelease, _ := version.LatestRelease()
+				fmt.Printf("WARN: you are using %s which is not the latest release (newest is %s). "+
+					"You can update it using:\n$ curl -sL http://git.io/get-ike | bash\n", v.Version, latestRelease)
```

Comment:
Technically the suggested script could be wrong, couldn't it. It assume ike is in the current dir? 

    curl .../get-ike |  bash -s — --version=v0.0.1 --dir=~`type ike*magic-parent-dir`

Reply:
Then maybe we should simply link to docs and fix that there?

Reply:
That would be a safe bet. Maybe link to release notes and include install notes there. In case some migration type of thing would be needed as well?

Reply:
b23fb82